scripts/cppcheck.sh

```diff
@@ -14,10 +14,12 @@ case $SCRIPT_DIR in
         ;;
 esac
 
+SRC_DIR=${1:-../src}
```

Comment:
I'm wondering if default value shouldn't be ``${SCRIPT_DIR}../src``  (and thus remove the ``${SCRIPT_DIR}/`` prefixing below) so that users can use their shell completion features to provide the directory in an intuitive way

That is from a build directory that is under the top directory, you'd do "../scripts/cppcheck.sh ../src/core/vector" for example

Reply:
You are right!

Reply:
fixed!